mmcv/cnn/resnet.py

```diff
@@ -255,14 +260,15 @@ def __init__(self,
                 dilation=dilation,
                 style=self.style,
                 with_cp=with_cp)
-            self.inplanes = planes * block.expansion
+            self.inplanes = planes * block.expansion  # type: ignore
             layer_name = f'layer{i + 1}'
             self.add_module(layer_name, res_layer)
             self.res_layers.append(layer_name)
 
-        self.feat_dim = block.expansion * 64 * 2**(len(stage_blocks) - 1)
+        self.feat_dim: int = 2**(len(stage_blocks) -
+                                 1) * 64 * block.expansion  # type: ignore
```

Comment:
```suggestion
        self.feat_dim = block.expansion * 64 * 2**(len(stage_blocks) - 1)  # type: ignore
```

Reply:
In this way, that line will be changed by `yapf` first, 
from:
```python
self.feat_dim = block.expansion * 64 * 2**(len(stage_blocks) - 1)  # type: ignore
```
into:
```python
self.feat_dim = block.expansion * 64 * 2**(
     len(stage_blocks) - 1)  # type: ignore
```
and won't pass the `mypy` test. 
Because `# type: ignore` should locate the same line as `block.expansion`, like:
```python
self.feat_dim = block.expansion * 64 * 2**(    # type: ignore
     len(stage_blocks) - 1)
```
So I wrote like this.

Reply:
Can we add a type hint for expansion to resolve the mypy error?
https://github.com/open-mmlab/mmcv/blob/bcd329143162da2026d05603fe2a0cab21502420/mmcv/cnn/resnet.py#L23
```python
expansion: int = 1
```
https://github.com/open-mmlab/mmcv/blob/bcd329143162da2026d05603fe2a0cab21502420/mmcv/cnn/resnet.py#L65
```python
expansion: int = 4
```

Or we can append `#noqa: E501` like

```python
self.feat_dim = block.expansion * 64 * 2**(len(stage_blocks) - 1)  # type: ignore  # noqa: E501
```

Reply:
Both methods are not working.
Still got the error from `mypy`:
```
mmcv/cnn/resnet.py:268: error: "Type[object]" has no attribute "expansion"
```

Reply:
Okay

Reply:
Okay